runtime/vm/ClassInitialization.cpp

```diff
@@ -418,7 +420,7 @@ classInitStateMachine(J9VMThread *currentThread, J9Class *clazz, J9ClassInitStat
 							U_8 *signatureChars = J9UTF8_DATA(signature);
 
 							PUSH_OBJECT_IN_SPECIAL_FRAME(currentThread, initializationLock);
-							J9Class *valueClass = internalFindClassUTF8(currentThread, signatureChars + 1, J9UTF8_LENGTH(signature) - 2, clazz->classLoader, J9_FINDCLASS_FLAG_THROW_ON_FAIL);
+							J9Class *valueClass = internalFindClassUTF8(currentThread, signatureChars, J9UTF8_LENGTH(signature), clazz->classLoader, J9_FINDCLASS_FLAG_THROW_ON_FAIL);
```

Comment:
once the Qtype is initialized, is there any need to initialize the ltype? If it has no members or state, perhaps it should just be automatically initialized once the qtype is initialized. In other words can they share intializations state

Reply:
Update the code to share the intializations state between L type and Q type.